### ISSUE TITLE
How to test in Local Environment

### ISSUE THREAD

**Issue by admin-dashboards** [admin-dashboards/nextjs-argon-dashboard#5]
Hello @Mortrest, 

The README in the `quick start` section should explain how to use the app in a local environment. 
Once the user got the secrets from the Oauth provider, he need an https connection for the SignIN callback. 

To solve this, a new target & dependency was added to the project: 

- `local-ssl-proxy` in `package.json`
- New execution target: `dev-ssl`
  - `next dev -p 3000 | local-ssl-proxy --source 5000 --target 3000` 

Using this set up, the flow works, but the app redirects the user forever to authenticate. 
Basically, the OAuth Sign In is not detected by the app. 

> Note: Chrome (incognito more) was used for tests. 

Please take a look.
ty!

**Comment by alivarastepour** [comment]
hello there.
no extra configuration is needed to test the OAuth in local env. setting the Authorization callback URL and Homepage URL in OAuth provider correctly will do the trick.
an example would be:
Homepage URL : http://localhost:3000/
Authorization callback URL : http://localhost:3000/api/auth/callback/${provider_name e.g github}
